assets/javascripts/audit_log.js

```diff
@@ -27,6 +63,21 @@ function loadAuditLogTable() {
                 }
             },
             { targets: 3, visible: false }, {
+                targets: 4,
+                render: function(data, type, row) {
+                    if (type === 'display') {
+                        var url;
+                        // Look for an id, and if we have one match it with an event type
+                        try {
+                            var url = getURLForType(row.event, JSON.parse(row.event_data));
+                            if (url) {
+                                return '<a class="audit_event_details" href="' + url + '">' + data + '</a>';
```

Comment:
Looks like we need to do some HTML-escaping here. (That one can not simply return a DOM element here is a really annoying limitation of DataTables.)